src/ert/scheduler/openpbs_driver.py

```diff
@@ -142,16 +142,35 @@ def __init__(
         self._finished_job_ids: Set[str] = set()
         self._finished_iens: Set[int] = set()
 
-        self._resources = self._resource_strings()
+        if self._num_nodes is not None and self._num_nodes > 1:
+            logger.warning(
```

Comment:
Isn't this reported already in here:
```python
 DeprecationInfo(
        keyword="QUEUE_OPTION",
        message="NUM_CPUS_PER_NODE as QUEUE_OPTION to Torque is deprecated and will removed in "
        "the future. Replace by NUM_CPU.",
        check=lambda line: "NUM_CPUS_PER_NODE" in line,
    ),
    DeprecationInfo(
        keyword="QUEUE_OPTION",
        message="NUM_NODES as QUEUE_OPTION to Torque is deprecated and will removed in "
        "the future. Replace by NUM_CPU on a single compute node.",
        check=lambda line: "NUM_NODES" in line,
    ),
```

Reply:
Correct. The config system will flash it in the GUI, while this will leave a trace in the jobqueue log.